components/03-organisms/header/staff-header/staff-header.html

```diff
@@ -1,42 +1,37 @@
-<div>
-    <header class="staff-header">
-        <div class="staff-header-content">
-            <div class="staff-header-quote">
-                <h2 class="staff-header-quote--text">"Wir lösen das persönlich."</h2>
-                <cite class="staff-header-quote--author" itemprop="name">Heribert Innoq</cite>
-            </div>
-            <img class="staff-header-image" itemprop="photo" src={context.app.uri("assets/example-content/heribert.jpg")} alt="Portrait von Heribert Innoq" />
-        </div>
-    </header>
-    <div class="staff-intro">
-        <h2 class="staff-intro--name" itemprop="name">Heribert Inooq</h2>
-        <h3 class="staff-intro--job-title" itemprop="jobTitle">Senior Consultant</h3>
-        <div class="staff-intro--text">
+<header class="staff-header">
+    <div class="staff-header-left"></div>
+    <img class="staff-header-image" itemprop="photo" src={context.app.uri("assets/example-content/heribert.jpg")} alt="Portrait von Heribert Innoq" />
+    <section class="staff-header-label">
+        <h2 class="staff-header-label__name" itemprop="name">Heribert Innoq</h2>
+        <h3 class="staff-header-label__job-title" itemprop="jobTitle">Senior Consultant</h3>
```

Comment:
Z-5 & 6: hier könnte das Markup noch etwas semantischer sein, schau mal bei der author-bio nach. Da hat Daphne das schön umgesetzt. :)

Reply:
Address-Tag ✅  `"name"` zu `"fullName"` geändert, `"givenName"` ist ja nur der Vorname, hab das in der Author-Bio auch angepasst.